egs/mls_english/ASR/local/utils/asr_datamodule.py

```
@@ -0,0 +1,253 @@
+
+    def test_dataloader(self) -> DataLoader:
+        """Create test dataloader."""
+        cuts = self.test_cuts()
+        return DataLoader(
+            self._create_dataset(cuts),
+            sampler=self._create_sampler(cuts, shuffle=False),
+            batch_size=None,
+            num_workers=self.args.num_workers,
+        )
+
+    @lru_cache()
```

Ruff (0.12.2)

Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)